.github/workflows/release.yml

```diff
@@ -0,0 +1,106 @@
+name: Release build
+env:
+  IMAGE_REGISTRY: "localhost:5000"
+  TAG: "latest"
+  DOCKER_ORG: enmasse
+  KUBERNETES_NAMESPACE: "enmasse-infra"
+  SYSTEMTEST_PROFILE: smoke
+  EXTERNAL_REGISTRY: "quay.io"
+  REGISTRY_USER: "obabec+actions"
+
+on:
+  push:
+    branches:
+      - master
+      - actions
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Export version
+      id: version
+      run: | 
+        echo "::set-env name=VERSION::$(grep "release.version" pom.properties| cut -d'=' -f2)"
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11.0.2
+
+    - name: setup-docker
+      uses: docker-practice/actions-setup-docker@0.0.1
+    
+    - name: Setup Ruby environment with ruby-build
+      uses: clupprich/ruby-build-action@v1.0.0
+
+    - name: Install dependencies
+      run: ./.github/scripts/install_dep.sh
+
+    - name: Minikube setup with registry
+      uses: obabec/minikube-setup-action@v0.0.4-alpha
+      with:
+        k8s-version: 1.15.0
+
+    - name: Build and run Unit tests
+      run : ./.github/scripts/build.sh
+
+    - name: Collect logs from Unit tests
+      run: ./.github/scripts/collectLogs.sh
+
+    - name: Smoke tests
+      run: ./.travis/test.sh        
+
+    - name: Collect logs from smoke tests
+      run: ./.github/scripts/collectLogs.sh
+
+    - name: Upload artifacts
+      if: failure()
+      uses: actions/upload-artifact@v1.0.0
+      with:
+        name: test-logs
+        path: artifacts
+
+    - name: Change image registry and re-tag images
+      run: |
+        ./.github/scripts/retag_images.sh
+        echo "::set-env name=IMAGE_REGISTRY::quay.io"
+
+    - name: Release binaries
+      if: success()
+      id: release
+      run: |
+        ./.github/scripts/release.sh
+        echo "::set-env name=TAG::$(grep "release.version" pom.properties| cut -d'=' -f2)"
+        echo ::set-output name=tag::$TAG
```

Comment:
I still thing retagging is not safe. If something goes wrong in the script, we may have a broken release template without knowing. I think we should just simplify this action and push images to quay and test with non-replaced templates.